modules/distr-tracing-tempo-install-overview.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: CONCEPT
+[id="distr-tracing-tempo-install-overview_{context}"]
+= {DTProductName} installation overview
+
+The steps for installing {DTProductName} are as follows:
+
+* Review the documentation and determine your deployment strategy.
```

Comment:
We only support one deployment strategy. I think we can remove this.

Reply:
Thank you, I'm removing this bullet point completely (if my understanding is correct):
```suggestion
```